### DISCUSSION TITLE
Upgrade the Sub for Mission/Sandbox Mode. Maybe even item quantity with the new item stacking.

### DISCUSSION THREAD

**Discussion by Kaillera** [FakeFishGames/Barotrauma#4688] [Ideas and suggestions]
Playing on higher difficulty on mission mode puts you at larger disadvantage than in campaign. The sub is completely non-upgraded, which makes it harder to withstand the new difficulty event set.

I think there should be a new slider for mission/sandbox mode that allows the player to select how upgraded the submarine is. Scale of 0 to 10. For upgrades that are only 5 levels, you divide that. Level 4/5 at 80% for instance.

Lastly I think it'd be nice to change the quantity of items in autofill so that it's more balanced toward the difficulty setting. You're going to definitely need a lot more medicines than what's given on autofill on average for difficulties above 30%.

**Comment by Ek-Videogames** [comment]
I weigh-in on the side of more customization.
Host GUI for setting specific upgrades for instance if a submarine desperately needs a certain type of upgrade like engine or crush depth (and hull obviously) but not things like reactor efficiency or fabricator upgrades if they don't need them.
